Mastodon/Scene/Onboarding/Login/MastodonLoginView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  MastodonLoginView.swift
+//  Mastodon
+//
+//  Created by Nathan Mattes on 10.11.22.
+//
+
+import UIKit
+import MastodonAsset
+import MastodonLocalization
+
+class MastodonLoginView: UIView {
+
+  // List with (filtered) domains
+
+  let titleLabel: UILabel
+  let subtitleLabel: UILabel
+  private let headerStackView: UIStackView
+  let searchTextField: UITextField
+  let tableView: UITableView
+  private var tableViewWrapper: UIView
+  let navigationActionView: NavigationActionView
+  var bottomConstraint: NSLayoutConstraint?
+
+  override init(frame: CGRect) {
+
+    titleLabel = UILabel()
+    titleLabel.font = MastodonLoginViewController.largeTitleFont
+    titleLabel.textColor = MastodonLoginViewController.largeTitleTextColor
+    titleLabel.text = L10n.Scene.Login.title
+    titleLabel.numberOfLines = 0
+
+    subtitleLabel = UILabel()
+    subtitleLabel.font = MastodonLoginViewController.subTitleFont
+    subtitleLabel.textColor = MastodonLoginViewController.subTitleTextColor
+    subtitleLabel.text = L10n.Scene.Login.subtitle
+    subtitleLabel.numberOfLines = 0
+
+    headerStackView = UIStackView(arrangedSubviews: [titleLabel, subtitleLabel])
+    headerStackView.axis = .vertical
+    headerStackView.spacing = 16
+    headerStackView.translatesAutoresizingMaskIntoConstraints = false
+
+    searchTextField = UITextField()
```

Comment:
Please set the keyboard type to `URL` and disable the auto-capitalization feature. Also, please make the search matching case insensitive. That will helps the user input Capitalized domain or copy from somewhere and continue the app suggestion.